src/natcap/invest/urban_nature_access.py

```diff
@@ -704,10 +930,18 @@ def _admin_level_supply_demand(
             greenspace_stats[feature_id]['sum'] /
             population_stats[feature_id]['sum'])
         feature.SetField(supply_sum_fieldname, avg_greenspace_supply_demand)
-        feature.SetField(
-            undersupply_fieldname, undersupplied_stats[feature_id]['sum'])
-        feature.SetField(
-            oversupply_fieldname, oversupplied_stats[feature_id]['sum'])
+        undersupply = undersupplied_stats[feature_id]['sum']
+        oversupply = oversupplied_stats[feature_id]['sum']
+        feature.SetField(undersupply_fieldname, undersupply)
+        feature.SetField(oversupply_fieldname, oversupply)
+        for pop_groupname in population_groupnames:
+            group_proportion = feature.GetField(f'pop_{pop_groupname}')
+            feature.SetField(
+                f'{oversupply_fieldname}_{pop_groupname}',
+                oversupply * group_proportion)
+            feature.SetField(
+                f'{undersupply_fieldname}_{pop_groupname}',
+                undersupply * group_proportion)
```

Comment:
As I'm understanding it, this method assumes that the different population groups are evenly distributed across the admin unit. Let's be sure to emphasize that in the user guide.

Reply:
Yes, that's true.  As Roy explained it over slack:

> This step is necessary because data on the general population numbers is often available at high resolution, whereas for other demographic statistics this data often is only available at neighborhood level at best.

I'll add a note to this to the UNA issue (#722 ) so I'm sure to tackle this in the UG.